Homework/Program.cs

```diff
@@ -4,7 +4,39 @@ internal class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello, World!");
+            {
+                for (int i = 0; i < 2; i++)
+                {
+
+
+                    Console.WriteLine("zadejte jméno a příjmení:");
+                    string? jmeno = Console.ReadLine();
+
+                    Console.WriteLine("zadejte vek:");
+                    string? vek = Console.ReadLine();
+
+                    Console.WriteLine("zadejte vahu v kilogramech:");
+                    string? vaha = Console.ReadLine();
+
+                    Console.WriteLine("zadejte výšku v centimetrech:");
+                    string? vyska = Console.ReadLine();
+
+                    {
+
+
+                        Console.WriteLine(jmeno + "je starý(a) " + vek + " let. Važí " + vaha + " kilogramů a je " + vyska + " cm vysoky/a ");
+                    }
+
+                    int vaha2 = int.Parse(vaha);
+                    int vyska2 = int.Parse(vyska);
+                    double BMI = vaha2 / ((vyska2 * 0.01) * (vyska2 * 0.01));
```

Comment:
Maly tip: tady by bylo efektivnejsi nejdriv vypocitat vysku v metrech do nejake promenee. Pocitac pak namisto 2 vypoctu bude delat jen jeden. Nejak takto:
```suggestion
                    double vyskaM = vyska2 * 0.01;
                    double BMI = vaha2 / (vyskaM * vyskaM);
```